### ISSUE TITLE
more firehose issues

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#281]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/281)) by Patrick Rowe (Bitbucket: [patrickrowe](https://bitbucket.org/%7Ba2fcb1b6-fada-4dd7-8471-07544dd69a68%7D/)).**

The original report had attachments: [impaled firehose.jpg](https://osrf-migration.github.io/drcsim-gh-pages/data/repositories/osrf/drcsim/issues/281/attachments/impaled_firehose.jpg)

----------------------------------------
1) I concur with the previous firehose issue that was raised where a gentle touch on any part of the hose starts the whole hose vibrating, which makes it move around and eventually fall off the table

2) My new issue is with the collision models. Quite frequently, i have been able to "impale" one of the fingers through what are supposedly solid parts of the hose. This includes the hose itself and the blue connector. Furthermore, once impaled, it is impossible to get it off the finger. I have attached pictures to show what is happening.

If this happens, there is no longer any way to manipulate the hose or connector.

Thank you,
Pat

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
The issue mentioned here should have been addressed in drcsim-nightly.  In drcsim 2.5.x, the Sandia hands have very soft contact parameters (kp = 1000 N/m), they've since been updated to something stiffer (kp = 100,000 N/m) in nightlies.  In addition, the fire hose geometry has been updated significantly as well (see [drcsim 2.6 in RoadMap](http://gazebosim.org/wiki/DRC/Roadmap)).


To tryout the nightly builds, do:
~~~
sudo apt-get remove drcsim gazebo sandia-hands osrf-common
sudo apt-get update
sudo apt-get install drcsim-nightly
~~~

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------

* changed state from "new" to "invalid"

fixed in drcsim 2.6
